howtos/range_queries.rst

```diff
@@ -142,11 +142,7 @@ prefixes.
 If you get the rules wrong, the QueryParser will raise a
 `QueryParserError`, which in production code you could catch and
 either signal to the user or perhaps try the query again without the
-`RangeProcessor` that tripped up:
-
-.. xapianrunexample:: search_ranges
-    :args: db 1000mm..
-    :shouldfail: 1
+`RangeProcessor` that tripped up.
```

Comment:
This seems the only change here that's at all contentious (the general plan to unify the output files seems a great idea).

I wonder if we should keep this, but unify the output by having exception handling in the example code.  Example code which totally omits error/exception handling is problematic.  The only real argument for it is that it makes the code a bit clearer, but there's a big downside when people model their code after the example code.

Thoughts?